test/Microsoft.Health.Dicom.Web.Tests.E2E/Rest/DataPartitionEnabledHttpIntegrationTestFixture.cs

```diff
@@ -15,7 +15,7 @@ public DataPartitionEnabledHttpIntegrationTestFixture()
         }
 
         protected DataPartitionEnabledHttpIntegrationTestFixture(string targetProjectParentDirectory)
-            : base(targetProjectParentDirectory, enableDataPartitions: true)
+            : base(targetProjectParentDirectory, enableDataPartitions: true, enableUpsRs: true)
```

Comment:
We dont need to set here. This fixture is for testing features where feature flag wont turned on by default

Reply:
Done